src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensEditsTests.cs

```diff
@@ -0,0 +1,307 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.LanguageServer.Handler.SemanticTokens;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Xunit;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.UnitTests.SemanticTokens
+{
+    public class SemanticTokensEditsTests : AbstractSemanticTokensTests
+    {
+        /*
+         * Markup for basic test case:
+         *     // Comment
+         *     static class C { }  
+         */
+        private static int[] StandardCase()
+            => new int[] {
+                // Line | Char | Len | Token type                                                            | Modifier
+                0,        0,     10,   SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Comment],  0, // '// Comment'
+                1,        0,     6,    SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Keyword],  0, // 'static'
+                0,        7,     5,    SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Keyword],  0, // 'class'
+                0,        6,     1,    SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Class],    (int)TokenModifiers.Static, // 'C'
+                0,        2,     1,    SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Operator], 0, // '{'
+                0,        2,     1,    SemanticTokensCache.TokenTypeToIndex[LSP.SemanticTokenTypes.Operator], 0, // '}'
+            };
```

Comment:
i don't love this.  i would recommend this actually be generated from a true call to classify that original markup  i.e. your test class can have:

```c#
private static readonly string s_standardCase = @"// Comment
...";
```

and you can refer to that in your tests and get this int-array form by calling your own classification methods.  i don't think you should hardcode this in in this form.